themes/digital.gov/src/scss/new/_card-news.scss

```diff
@@ -1,38 +1,36 @@
 @use "uswds-core" as *;
 
-.card.card-elsewhere {
+.card.card-news--external {
   grid-template-areas:
     "c-media c-content"
     "c-meta c-meta";
   grid-template-columns: #{units(7)} 1fr;
 
   .card__summary {
-    @include u-margin-top(0);
+    margin-top: 0;
 
     p {
-      @include u-line-height("sans", 3);
-      @include u-font("sans", "md");
-      @include u-text("normal");
-
+      font-family: font-family("sans");
+      font-weight: font-weight("normal");
+      line-height: line-height("sans", 3);
       max-width: none;
     }
   }
 
   .card__source {
-    @include u-margin-top("05");
-    @include u-display("block");
+    display: block;
+    margin-top: units("05");
 
     a {
-      @include u-color("blue-50v");
+      color: color("blue-50v");
```

Comment:
```suggestion
      color: color("primary");
```

Alternatively we could remove this and rely on the default link styles.

**Before/after**
![image](https://github.com/GSA/digitalgov.gov/assets/3385219/685a7ac6-0524-414c-aaae-1013496ec6e2)

Reply:
Removed